client/ayon_deadline/plugins/publish/aftereffects/submit_aftereffects_deadline.py

```diff
@@ -20,7 +20,7 @@ class DeadlinePluginInfo():
     ProjectPath: str = field(default=None)
     AWSAssetFile0: str = field(default=None)
     Version: str = field(default=None)
-    MultiProcess: str = field(default=None)
+    MultiProcess: bool = field(default=None)
```

Comment:
Should this remain `: str` for Deadline to interpret correctly? If so, I just need to fix how we're setting it instead.

Reply:
This is fine, it gets converted automatically.